src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int bound = 100;
```

Comment:
constants should be static final and have names in UPPER_CASE
```suggestion
    private static final int BOUND = 100;
```